pom.xml

```diff
@@ -1,46 +1,53 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-
-  <groupId>com.donation</groupId>
-  <artifactId>donation-app</artifactId>
-  <version>0.1.0-SNAPSHOT</version>
-  <packaging>pom</packaging>
-
-  <modules>
-    <module>donation-core</module>
-    <module>donation-repository</module>
-    <module>donation-web</module>
-  </modules>
-
-  <properties>
-    <java.version>1.8</java.version>
-    <spring.boot.version>2.1.1.RELEASE</spring.boot.version>
-    <swagger-version>2.9.2</swagger-version>
-    <servlet-api>2.5</servlet-api>
-    <jackson-version>2.9.5</jackson-version>
-  </properties>
-
-  <parent>
-    <groupId>org.springframework.boot</groupId>
-    <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.1.1.RELEASE</version>
-  </parent>
-
-  <dependencies>
-    <!--Spring boot-->
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-web</artifactId>
-      <version>${spring.boot.version}</version>
-    </dependency>
-
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-security</artifactId>
-      <version>${spring.boot.version}</version>
-    </dependency>
-
-  </dependencies>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.donation</groupId>
+    <artifactId>donation-app</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>donation-core</module>
+        <module>donation-repository</module>
+        <module>donation-web</module>
+    </modules>
+
+    <properties>
+        <java.version>1.8</java.version>
+        <spring.boot.version>2.1.1.RELEASE</spring.boot.version>
+        <swagger-version>2.9.2</swagger-version>
+        <servlet-api>2.5</servlet-api>
+        <jackson-version>2.9.5</jackson-version>
+    </properties>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.1.1.RELEASE</version>
+    </parent>
+
+    <dependencies>
+        <!--Spring boot-->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+            <version>${spring.boot.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-security</artifactId>
+            <version>${spring.boot.version}</version>
+        </dependency>
+
+        <!-- Reactor -->
+        <dependency>
+            <groupId>io.projectreactor</groupId>
```

Comment:
I thought Reactive programming are included in java 9, do we really need this dependency ?

Reply:
I tried switching to Java 10 but i still needed the dependency

Reply:
Ok I'll check it later